### ISSUE TITLE
Alert sound independency

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#369]
```
Basically, I would like to see the alert sound not beeing one of the windows 
themed sounds and requiring the sounds in windows to be turned on, but rather 
independant and customizable from the client settings instead.
```

Original issue reported on code.google.com by `adrianlund` on 20 Jan 2013 at 11:32

**Comment by mintty** [comment]
Adding some configuration options to choose among system sounds, or to select a plain frequency sound. Not yet what you wanted, though...

**Comment by mintty** [comment]
A wave sound file can now be configured in ~/.minttyrc – not yet in the Options menu, but sufficient to close the issue, I think.

**Comment by mintty** [comment]
Released in 2.2.3.
GUI configuration option may follow...

**Comment by mintty** [comment]
GUI configuration of wav file beep added in 2.2.4.
